File: src/test/java/com/aymanosman/racket/plugin/RacketAnnotatorTest.java
package org.racket.lang.core;

import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase;

public class RacketAnnotatorTest extends LightJavaCodeInsightFixtureTestCase {
    @Override
    protected String getTestDataPath() {
        return "src/test/testData";
    }

    public void testAnnotation() {
        myFixture.configureByFiles("annotator-test-data.rkt");

        myFixture.checkHighlighting(false, true, true, true);
    }
}

File: src/test/kotlin/org/racket/lang/core/RacketAnnotatorTest.kt
package org.racket.lang.core

import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase

class RacketAnnotatorTest : LightJavaCodeInsightFixtureTestCase() {
    override fun getTestDataPath(): String {
        return "src/test/testData"
    }

    fun testAnnotation() {
        myFixture.configureByFiles("annotator-test-data.rkt")
        myFixture.checkHighlighting(false, true, true, true)
    }
}
